### PR TITLE
Updates INFO fields about diskless loading progress and maxclients

### PR DESCRIPTION
**Pull request by itamarhaber**
Ref: https://github.com/redis/redis/pull/7981
Ref: https://github.com/redis/redis/pull/7979

### CODE REVIEW

File: commands/info.md [hunk 1]
```diff
@@ -277,7 +279,11 @@ If the instance is a replica, these additional fields are provided:
 
 If a SYNC operation is on-going, these additional fields are provided:
 
+*   `master_sync_total_bytes`: Total number of bytes that need to be transferred
+*   `master_sync_read_bytes`: Number of bytes already transferred
 *   `master_sync_left_bytes`: Number of bytes left before syncing is complete
+*   `master_sync_perc`: The percentage `master_sync_read_bytes` from 
+    `master_sync_total_bytes`
```
Comment (oranagra) [inline commands/info.md:286]:
```suggestion
*   `master_sync_perc`: The percentage `master_sync_read_bytes` from 
    `master_sync_total_bytes` or an approximation that uses `loading_rdb_used_mem` when `master_sync_total_bytes` is 0
```

**Comment by oranagra** [comment]
@itamarhaber can you make one for https://github.com/redis/redis/pull/7979 too? (or add it to this PR?)

File: commands/info.md [hunk 2]
```diff
@@ -73,6 +73,9 @@ Here is the meaning of all fields in the **clients** section:
 
 *   `connected_clients`: Number of client connections (excluding connections
      from replicas)
+*   `cluster_connections`: An approximation of the number of sockets used by the
+     cluster's bus
+*   `maxclients`: The value of the `maxclients` configuration directive
```
Comment (oranagra) [inline commands/info.md:78]: do we wanna mention that it should be matched with the sum of connected_clients, connected_slaves, and cluster_connections (to be able to tell how much remain available)?

Comment (itamarhaber) [reply commands/info.md:78]: That's a good idea unless perhaps we want to add a new `remaining_connections` field that reflects this metric?

Comment (oranagra) [reply commands/info.md:78]: it was suggested, but IIRC we decided to just reflect raw metrics and let the user do the math.

Comment (itamarhaber) [reply commands/info.md]: Please review the addition made in 6a754bb

**Review by oranagra — APPROVED** [review]
let's wait to see that the changes to redis are merged before merging this 8-)

**Pull request merged**